### PR TITLE
feature: support empty prefix

### PR DESCRIPTION
**Pull request by shaharkazaz**
What has been done:
* Add support for passing an empty string as a prefix (no prefix)
* fix generators specs + add specs for the new feature
* fix missing svgoConfig message ( was outputting `[object Object]`);

### CODE REVIEW

File: src/lib/generators/code-snippet-generators.spec.ts [hunk 1]
```diff
@@ -1,19 +1,96 @@
...
 describe('Generators', () => {
-  it('should return the correct interface definition', () => {
-    const interfaceName = 'TestIcons';
-    const typeName = 'myAwesomeIcons';
-    const expectedDefinition = `export interface ${interfaceName}{
-        name: ${typeName};
+  describe('generateInterfaceDefinition', () => {
+    it('should return the correct interface definition when passing generateType', () => {
+      const options = { generateType: true, typeName: 'AppIcons', interfaceName: 'IconInterface' } as
+        | MultiFileConvertionOptions
```
Comment (kreuzerk) [inline src/lib/generators/code-snippet-generators.spec.ts:14]: I think the | is not needed

File: src/lib/generators/code-snippet-generators.spec.ts [hunk 2]
```diff
@@ -24,8 +101,24 @@ describe('Generators', () => {
 
     const expectedSVGConstant = `export const ${variableName}: ${interfaceName} = {
                 name: '${filenameWithoutEnding}',
-                data: '${data}'
+                data: \`${data}\`
```
Comment (kreuzerk) [inline src/lib/generators/code-snippet-generators.spec.ts:104]: \` should not be needed, ${data} should be enough

File: src/lib/generators/code-snippet-generators.ts [hunk 3]
```diff
@@ -21,28 +21,28 @@ export const generateInterfaceDefinition = (
 };
 
 export const generateTypeDefinition = (
-  convertionOptions: MultiFileConvertionOptions | SingleFileConvertionOptions,
+  conversionOptions: MultiFileConvertionOptions | SingleFileConvertionOptions,
```
Comment (kreuzerk) [inline src/lib/generators/code-snippet-generators.ts:24]: Nice catch - actually we should also rename `MultiFileConvertionOptions` and `SingleFileConvertionOptions`. But thats something for another PR.

File: src/lib/generators/code-snippet-generators.spec.ts (hunk 2 above)
Comment (shaharkazaz) [reply src/lib/generators/code-snippet-generators.spec.ts:104]: I'm comparing the strings, and the output does have backticks, without it the spec fails.

File: src/lib/generators/code-snippet-generators.spec.ts (hunk 1 above)
Comment (shaharkazaz) [reply src/lib/generators/code-snippet-generators.spec.ts:14]: correct, I just copied the type from the signature 😄

File: src/lib/generators/code-snippet-generators.spec.ts (hunk 2 above)
Comment (kreuzerk) [reply src/lib/generators/code-snippet-generators.spec.ts:104]: Okay - makes sense

**Pull request merged**